package.json

```diff
@@ -78,9 +78,11 @@
     "etcher-latest-version": "^1.0.0",
     "file-type": "^4.1.0",
     "flexboxgrid": "^6.3.0",
+    "gpt": "^1.0.0",
     "immutable": "^3.8.1",
     "lodash": "^4.5.1",
     "lzma-native": "^1.5.2",
+    "mbr": "^1.1.1",
```

Comment:
Minor point, but would it be worth creating a higher-level module which requires `mbr` and `gpt` (and then does much of the work that you're doing here), and then have Etcher depend on that module?